develop/how-to-report-issues/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "How to Report an Issue"
+---
+
+To create a new issue, you must have a [GitHub](https://github.com/) account. If you really can't open or use one, consider posting to [wx-users mailing list](/support/mailing-lists/), but creating the issue directly is strongly preferred.
+
+Please ***search*** to check if the bug or proposed enhancement is already submitted before reporting it. If you are not sure that you encountered the same bug, please open a new ticket and add mutual links to both tickets (`#1234` is automatically linked). 
```

Comment:
Is it correct to still use the word "ticket" instead of "issue"?
```suggestion
Please ***search*** to check if the bug or proposed enhancement is already submitted before reporting it. If you are not sure that you encountered the same bug, please open a new issue and add mutual links to both issues (`#1234` is automatically linked). 
Also, is not one mention in either issue enough, the other will get a reference automatically?
```

Reply:
This is another place where I've just copied the existing text, only reformatting in Markdown (using semi-automated substitutions). FWIW I think we shouldn't tell people opening new bugs at all if they suspect it's the same bug anyhow.

Reply:
Sorry, I have missed "There are certainly other improvements that could be made, but for now I've done the strict minimum to avoid having dangling/out of date links. Please let me know if I blundered anywhere." in the PR text.